pkg/sql/logictest/testdata/logic_test/partial_index

```diff
@@ -1,3 +1,4 @@
+# LogicTest: !3node-tenant
```

Comment:
This is required for the `SHOW CREATE TABLE...` statements I added. Without it the build fails with `relation "system.zones" does not exist`. Why?